### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Billy0220** [community/community#46495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Lists is a good way to categorize the starred repositories. However, once I have more starred repositories, the Star would be chaos (I think). What if the Lists is like the folder to place the starred repositories. I mean once I put one starred repository in my own category, the starred repository in Stars would be removed (to the category in Lists) or be hiddend. I think it would be better

**Comment by bulutuu** [comment]
Derdo

**Comment by Ada1433** [comment]
Yes

**Comment by Chuiunbg** [comment]
Meme
